pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -336,14 +336,15 @@ func (o *InstallOptions) copyContent(ctx context.Context) error {
 func (o *InstallOptions) Run(ctx context.Context) error {
 	var eventTarget *corev1.ObjectReference
 
-	if err := retry.RetryOnConnectionErrors(ctx, func(context.Context) (bool, error) {
+	err := retry.RetryOnConnectionErrors(ctx, func(context.Context) (bool, error) {
 		var clientErr error
 		eventTarget, clientErr = events.GetControllerReferenceForCurrentPod(o.KubeClient, o.Namespace, nil)
 		if clientErr != nil {
 			return false, clientErr
```

Comment:
you shouldn't fail on this

Reply:
i'm trying to get it a chance to get real reference in case this is really connection error...